Logging to stdout and better errors (#186)
nf-core/epitopeprediction: CHANGELOG.md

### `Added`

- [#180](https://github.com/nf-core/epitopeprediction/pull/180) - Add support for `VEP` annotated VCF files [#172](https://github.com/nf-core/epitopeprediction/issues/172)

> - [#186](https://github.com/nf-core/epitopeprediction/pull/180) - Logging of epaa.py script also to stdout and added sys.exit error messages.

Reviewer: ```suggestion
- [#186](https://github.com/nf-core/epitopeprediction/pull/186) - Log messages from `epaa.py` script to stdout and provide `sys.exit` error messages.
```
